### ISSUE TITLE
Biome support?

### ISSUE THREAD

**Issue by Zottelchen** [GregoryAM-SP/The-Minecraft-Overviewer#19]
Hi, quick question. 

Do I somehow need to enable biome support for this clone? I am trying to render a 1.17 map while using an old config. The result looks like this:
![z0](https://github.com/GregoryAM-SP/The-Minecraft-Overviewer/assets/5148555/eaa54248-f1d7-406c-8d6d-15b053ffc831)

A previous version (rendered with the OG Overviewer) of the old 1.16 map is properly biome colored:
![smallmap](https://github.com/GregoryAM-SP/The-Minecraft-Overviewer/assets/5148555/3ab4c000-d30f-4a7e-a125-5ab03b0ce7c9)

**Comment by Gregory-AM** [comment]
Hey there, @Zottelchen!
I'm downloading the map now, and double checking to see if there are any issues while rendering it.

Do you use a configuration file while rendering this map?

**Comment by Zottelchen** [comment]
Yeah, my configuration file is quite big as I wrote filters for a lot of stuff: https://map.drehmal.cyou/drehmal_render.conf

Make sure to download the Beta version 2.2: https://www.drehmal.net/apotheosisbeta / [direct link@mega.nz](https://mega.nz/file/XIxl3aQA#hEjQOawbiTcA-IGi9zpp9y6QXv-AgpJ-INaR_g_BE4Y).

**Comment by Gregory-AM** [comment]
I'm now downloading the Beta version, and I just downloaded the configuration.
I'll have to get back to you on this issue after a render.

Thank you for sharing the configuration and which world.

**Comment by Gregory-AM** [comment]
I'm going to get back to this issue over the weekend.
With how long the render is taking, I'm looking at a 8+ hour render time.
I'm planning on making multiple renders with your configuration and modifying it to see if anything changes.

**Comment by Gregory-AM** [comment]
@Zottelchen,
Do you know of the last version of Overviewer that worked with rendering the biome colors?
Was it the last update that the original developers released, 1.19.10?

**Comment by Zottelchen** [comment]
> @Zottelchen, Do you know of the last version of Overviewer that worked with rendering the biome colors? Was it the last update that the original developers released, 1.19.10?

Probably not, I released that rendered map in November 2020.

**Comment by Gregory-AM** [comment]
@Zottelchen, I would recommend using the latest version of Overviewer that the original developers released on Overviewer.org, but keep the current version you're working with as a just in case it fails.

I tested it with the biome colors and it works, I'm not sure about the entities though as I'm sure the world has to be loaded at least once, and I'm having issues getting it to work for me.

---

I am going to see what went wrong when adding biome coloring for 1.20 and try to add backwards compatibility, but this will take time and I'm not sure how long it will take; as I'm planning on recloning Minecraft-Overviewer and slowly adding each change I've made to debug this issue since I know it's important to make sure Overviewer is compatible with every version of Minecraft.

**Comment by JoshuaMarti** [comment]
@Zottelchen, were you able to figure it out? I just checked your map page linked and it looks like things are properly colored.

**Comment by Gregory-AM** [comment]
> @Zottelchen, were you able to figure it out? I just checked your map page linked and it looks like things are properly colored.

I think that's still the older version of his map.
Sadly I haven't gotten around to this as I've been transitioning jobs, and doing things outside of development.

I tried pulling the Overviewer repo and comparing it to the successor and haven't been able to find what's causing this issue. I did this about a month ago.

**Comment by Zottelchen** [comment]
@JoshuaMarti @Gregory-AM 
[It is actually the updated version of the map](https://dl.drehmal.cyou/2.2/z3.png) (e.g. look at the northern/snow part, the volcano looks different), but for the time being I've just used the old Overviewer (and for POIs I currently use basically a [hardcoded variant of my config](https://github.com/Zottelchen/poigen)).
